core/src/consensus.rs

```diff
@@ -30,11 +30,11 @@ use std::{
 };
 use thiserror::Error;
 
-#[derive(PartialEq, Clone, Debug)]
+#[derive(PartialEq, Clone, Debug, AbiExample)]
 pub enum SwitchForkDecision {
     SwitchProof(Hash),
-    NoSwitch,
```

Comment:
I've always been bothered with this naming `NoSwitch` everytime I see this on logs like this:

- At the first glance, it sounds something bad happened due to the connotation of `No`. (it's not negating errorous thing like `Error` as in `NoError`) But this is in fact pretty normal enum variant.
- Then, my next wondering is: If there isn't something called `Switch`, what is this exactly?

So, I'm proposing here to rename to `SameFork`:

- `Same` sounds good thing or boring/usual thing at least.
- It doesn't introduce new term; it borrows the `Fork` term in the enum name. So this doesn't introduce any additional cognitive load.
 - `SameFork` adequately implies we're voting on same fork without `Switch` if the reader has basic understaing of validator's voting behavior. 
- From my understanding, I think `SameFork` correctly represents what this variant is meant for.


@carllin Time for bike shedding? ;)

Reply:
Sounds fine by me!